src/include/daos_srv/evtree.h

```diff
@@ -579,40 +589,39 @@ int evt_find(daos_handle_t toh, const struct evt_filter *filter,
 int evt_debug(daos_handle_t toh, int debug_level);
 
 enum {
+	/** Return extents visible in the search rectangle */
+	EVT_ITER_VISIBLE	= (1 << 0),
+	/** Add fully or partially covered extents to EVT_ITER_VISIBLE */
+	EVT_ITER_COVERED	= (1 << 1) | EVT_ITER_VISIBLE,
```

Comment:
I changed it when I added the EVT_ITER_DELETED flag.   Since we don't actually have any users of the COVERED only option, it just seemed simpler to only be additive.  It's probably not as attractive without EVT_ITER_DELETED but doesn't hurt anything and reduces the test burden, particularly for things we don't even use.

Reply:
@wangdi1 that is a reasonable point.   Perhaps we could just use EVT_ITER_SORTED